contrib/cmd/memfd-bind/README.md

```diff
@@ -1,8 +1,8 @@
 ## memfd-bind ##
 
-`runc` normally has to make a binary copy of itself (or of a smaller helper
-binary called `runc-dmz`) when constructing a container process in order to
-defend against certain container runtime attacks such as CVE-2019-5736.
+`runc` normally has to make a binary copy of itself when constructing a
+container process in order to defend against certain container runtime attacks
+such as CVE-2019-5736.
```

Comment:
Ah, I forgot to document how the new overlay mode works in #4448. I'll open a separate PR for that.

Reply:
this is more of a discussion, so with the overlay change i guess memfd-bind is no longer needed? :thinking:

Reply:
This is being discussed in https://github.com/opencontainers/runc/issues/4450#issuecomment-2448933143. Short answer: it has fewer upsides now and there is a fairly strong case for removing it.